### DISCUSSION TITLE
Plugin_202203222243884

### DISCUSSION THREAD

**Discussion by giscus** [tiddly-gittly/TiddlyWiki-CPL#79] [Plugins]
# Plugin_202203222243884

https://tw-cpl.netlify.app/

**Comment by Zacharia2** [comment]
TZN2018：https://github.com/SonephetR2/tiddly-aliases   这个经测试非常好用,和我设想的一样,只要加个"aliases"字段,填上别名,在其他页面正常的链接[[别名]]就可直接链接了,如果有两个别名只需用空格分开。

如果不是一个插件的话,建议替换原两那个没说明的（指的这个https://github.com/mklauber/tiddly-aliases）

**Reply by Zacharia2** [reply]
这个是未合并到主仓库但也很好用的aliases插件地址：https://github.com/SonephetR2/tiddly-aliases

**Comment by Zacharia2** [comment]
基于 JS 的 Wikipedia 样式别名和重定向的实现，用于不存在的条目标题。此插件允许您使用“aliased”字段设置条目别名。然后，所有别名也将链接到原条目。这对于您希望多个术语指代同一事物的情况很有用，例如当一个角色有多个名称时。
使用方法：安装插件后在需要别名的条目中加"aliases"字段,字段值为别名,多个别名用空格分开,链接方式只支持[[条目别名]],不支持宏链接和 transclusion 的方法{{条目别名}}
